config/environments/production.rb

```diff
@@ -127,7 +127,7 @@
   protocol = ENV["APP_PROTOCOL"] || "http://"
 
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.perform_deliveries = true
+  config.action_mailer.perform_deliveries =  ENV["SMTP_PASWORD"].present? || ENV["SENDGRID_API_KEY"].present?
```

Comment:
non-blocking: It looks like there's a tiny bit of repetition between this line and [`SiteConfig.smtp_enabled?`](https://github.com/forem/forem/pull/13650/files#diff-68da686a0f7348bf9a2c59e669043f21019ab8dfb6363cea6daa534c5c8296d7R233)

We can't use `SiteConfig`'s query here because it's possibly not ready/loaded at this stage? (I'm not actually sure and didn't test it).

Can we use this `perform_deliveries` configuration in `SiteConfig.smtp_enabled?`and limit the number of places we need to reference the environment variables?

Reply:
I didn't test it either but that's also my assumption that the configuration in `production.rb` would be loaded first.

Reply:
Updated!